packages/@remirror/extension-annotation/src/positioners.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  Coords,
+  emptyVirtualPosition,
+  hasStateChanged,
+  Positioner,
+} from 'remirror/extension/positioner';
```

Comment:
Internally you should use the `@remirror/extension-positioner`.

```suggestion
} from '@remirror/extension-positioner';
```